src/views/Settings.vue

```diff
@@ -309,10 +331,34 @@ export default defineComponent({
         }
       } catch(err) {
         logger.error('Failed to fetch current facility details', err)
-      } finally {
-        // declaration of isStoreFulfilmentTurnedOn in lifecycle hooks always
-        // returns 'true' because of the != 0 condition, hence, updating it here
-        this.isStoreFulfilmentTurnedOn = this.currentFacilityDetails?.maximumOrderLimit != 0
+      }
+    },
+    async getCurrentOrdersCount() {
+      let resp: any;
+      try {
+        resp = await UserService.getCurrentOrdersCount({
+          "entityName": "FacilityOrderCount",
+          "inputFields": {
+            "facilityId": this.currentFacility.facilityId
+          }
+        })
+        if (!hasError(resp) && resp.data.count) {
+          this.currentFacilityDetails.orderCount = resp.data.count
+        } else {
+          throw resp.data
+        }
+      } catch(err) {
+        console.error("Failed to fetch total orders count", err);
+      }
+    },
+    updateOrderLimitType() {
+      this.fulfillmentOrderLimit = this.currentFacilityDetails?.maximumOrderLimit
+      if (this.currentFacilityDetails?.maximumOrderLimit === 0) {
```

Comment:
As we have already stored the value of `this.currentFacilityDetails?.maximumOrderLimit` in `fulfillmentOrderLimit`, we can use it instead.